src/components/button/button.css

```
@@ -2,7 +2,7 @@
   border-radius: 20px;
   font-size: 1.2rem;
   color: white;
-  background: #016b6b;
+  background: var(--color-secondary);
```

I think this should be 
```suggestion
  background: var(--color-primary);
```